flux/base/00-remove-objects-patch.yaml

```diff
@@ -0,0 +1,49 @@
+$patch: delete
```

Comment:
Why are the patch files indexed?

Reply:
to keep them in order can remove this in next PR.